old-tutorials-in-development/tutorials-in-development/R/dc-spatio-temporal-intro/04-intro-CRS-projection.Rmd

```diff
@@ -116,35 +117,32 @@ To define the location of something we often use a coordinate system. This syste
 consists of an X and a Y value, located within a 2 (or more) -dimensional space.
 
 <figure>
-	<a href="http://open.senecac.on.ca/clea/label/projectImages/15_276_xy-grid.jpg">
-	<img src="http://open.senecac.on.ca/clea/label/projectImages/15_276_xy-grid.jpg"></a>
+	<a href="https://raw.githubusercontent.com/NEONScience/NEON-Data-Skills/dev-aten/graphics/vector-general/2D_Coordinates.jpg">
+	<img src="https://openpress.usask.ca/introgeomatics/chapter/frames-of-reference-and-coordinate-systems/" alt="An X, Y grid with points expressed as ordered pairs."></a>
```

Comment:
@donal-at-NEON: I downloaded and added an image here. Did I do this right? Should the `href` be the same as the `img src` or can the be different like what I did? The image was included in the commit and PR so hopefully that came through!

Reply:
I wasn't 100% sure my first image was open access so I went ahead and found pretty much the same image that's in the public domain. I updated the github link and pushed the new image into this PR. I deleted the old image from the directory so that doesn't confuse anyone.

Reply:
Perfect, thank you!